replica/table.cc

```diff
@@ -405,9 +407,17 @@ table::do_add_sstable(lw_shared_ptr<sstables::sstable_set> sstables, sstables::s
     return new_sstables;
 }
 
-void table::add_sstable(sstables::shared_sstable sstable) {
-    _main_sstables = do_add_sstable(_main_sstables, std::move(sstable), enable_backlog_tracker::yes);
-    refresh_compound_sstable_set();
+void compaction_group::add_sstable(sstables::shared_sstable sstable) {
+    _main_sstables = _t.do_add_sstable(_main_sstables, std::move(sstable), enable_backlog_tracker::yes);
+    _t.refresh_compound_sstable_set();
```

Comment:
This violates layering.
If that's (hopefully) temporary just for doing an incremental conversion of the code,
let's add a FIXME comment explaining and documenting it.

Reply:
Yes, I really dislike this too. When writing this series, I tried to limit the amount of changes, and this was one of them. But I think I can move _t.refresh_compound_sstable_set() to be called by table itself, rather than relying on compaction group to do so.